pkg/apis/psmdb/v1/psmdb_types.go

```diff
@@ -802,6 +811,35 @@ func (p PITRSpec) Disabled() PITRSpec {
 	return p
 }
 
+type BackupTimeouts struct {
+	Starting *uint32 `json:"startingStatus,omitempty"`
+}
+
+type BackupOptions struct {
+	OplogSpanMin float64           `json:"oplogSpanMin"`
+	Priority     map[string]string `json:"priority,omitempty"`
+	Timeouts     *BackupTimeouts   `json:"timeouts,omitempty"`
+
+	// TODO: Do we need to set these here since we have them in BackupTaskSpec?
+	// Compression      compress.CompressionType `json:"compression,omitempty"`
+	// CompressionLevel *int                     `json:"compressionLevel,omitempty"`
```

Comment:
LGTM, but need to remove the comment

Reply:
[655ecb2](https://github.com/percona/percona-server-mongodb-operator/pull/1490/commits/655ecb2f81bbeb9b56869a7cb6939c2d1b9f64bb)